pages/categoryPage.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+import { Selector } from 'testcafe';
+
+class CategoryPage {
+    constructor() {
+        this.categoryTitle = Selector('.page_contenthub_content .pageheader');
+        this.topSellersTab = '#tab_select_TopSellers';
+        this.availableTopSellersGames = '#TopSellersRows a';
+        this.gameTitle = '.tab_item_name';
+        this.gameCurrentPrice = '.discount_final_price';
+        this.gameInitialPrice = '.discount_original_price';
+        this.gameDiscount = ' .discount_pct';
+    }
+
+    async getTitle() {
+        return await this.categoryTitle.innerText;
+    }
+
+    async findCheapestGame() {
+        const currentPriceSelector = (i) => Selector(this.availableTopSellersGames).nth(i).find(this.gameCurrentPrice);
+        const initialPriceSelector = (i) => Selector(this.availableTopSellersGames).nth(i).find(this.gameInitialPrice);
+        const gamesAmount = await Selector(this.availableTopSellersGames).count;
```

Comment:
currently the function works only for Top Sellers games. it would be useful to add a parameter (to specify the tab name) to make the function more reusable